storage/mounts_test.go

```diff
@@ -21,3 +25,92 @@ func TestValidateMountDirectory(t *testing.T) {
 		assert.Lenf(t, errs, tc.errorCount, "directory '%s' does not generate the expected error count", tc.directory)
 	}
 }
+
+var expectedCommandResp = "done"
+
+func testMountFuncHelper(t *testing.T, mountName, expectedCommand string, mount Mount,
+	mountFunc func(mp MountPoint, mount Mount) (string, error)) {
```

Comment:
please move mountFunc to beginning of the arg list, after `t`, because it's the most important thing that this function does. During reviewing tests everyone should read it first, not the assertion arguments.